client/src/modules/chapters/graphql/queries.ts

```diff
@@ -51,7 +52,7 @@ export const CHAPTERS = gql`
       description
       logo_url
       banner_url
-      events {
+      events(first: 2) {
```

Comment:
I thought this would work, following the docs
https://graphql.org/learn/pagination/#slicing

Didn't work tho, got unknown argument which hinted that gql doesn't have that function in it.

here is the error.

>  Error 0: Unknown argument "first" on field "ChapterCardRelations.events".

```ts
      events(first: 2) {
```

here is an image of the error 

![image](https://user-images.githubusercontent.com/88248797/208916722-6479ac03-4143-4485-b2c1-0b006aa0412d.png)